drivers/clock_control/CMakeLists.txt

```diff
@@ -40,3 +40,8 @@ else()
   zephyr_sources_ifdef(CONFIG_SOC_SERIES_STM32WLX clock_stm32l4_l5_wb_wl.c)
 endif()
 endif()
+
+if(CONFIG_SOC_SERIES_AGILEX)
+  zephyr_sources(clock_agilex_ll.c)
+  zephyr_sources(clock_agilex.c)
```

Comment:
Why not using `zephyr_sources_ifdef`?

Reply:
Noted. I will update to

zephyr_sources_ifdef(CONFIG_SOC_SERIES_AGILEX clock_agilex_ll.c)
zephyr_sources_ifdef(CONFIG_SOC_SERIES_AGILEX clock_agilex.c)